server/controllers/controller.js

```diff
@@ -1,6 +1,41 @@
 const Model = require('../db-models/database');
 const axios = require('axios');
 
+exports.createGallery = async ({ headerImage, mainImages, moreLikeThisImages }) => {
+  let count = await Model.countDocuments({}, function (err, count) {
+    if (err) {
+        console.log(err);
+    } else {
+        return count++;
+    }
+  });
+
+  count++;
+
```

Comment:
Im curious why count is incremented twice in the file? I wonder if there is a way to simplify this.

Reply:
good catch. i need to remove the first increment. the first increment does not actually increment count which is why i had to add a second increment. i'll remove the first increment